HDFS-16026. Restore cross platform mkstemp

GauthamBanasandra opened the pull request:
* This reverts commit aed13f0f42fefe30a53eb73c65c2072a031f173e.

GauthamBanasandra committed:
HDFS-15971. Make mkstemp cross platform

This reverts commit aed13f0f42fefe30a53eb73c65c2072a031f173e.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 32s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  37m 26s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   3m  1s |  |  trunk passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   3m  2s |  |  trunk passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  mvnsite  |   0m 31s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  59m  1s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 15s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 51s |  |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  cc  |   2m 51s |  |  the patch passed  |
| +1 :green_heart: |  golang  |   2m 51s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 51s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 55s |  |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  cc  |   2m 55s |  |  the patch passed  |
| +1 :green_heart: |  golang  |   2m 55s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 55s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 17s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  15m  8s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  40m 52s |  |  hadoop-hdfs-native-client in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 33s |  |  The patch does not generate ASF License warnings.  |
|  |   | 124m 47s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense compile cc mvnsite javac unit codespell golang |
| uname | Linux 16fcdd187715 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / be7dfcb805b97e1f08107b36b8ac51381885278d |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/3/testReport/ |
| Max. process+thread count | 539 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/3/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Add stage for Centos 7

* Build the change on
  Centos 7 docker using
  the Dockerfile for the
  same.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  |   5m  8s |  |  Docker failed to build yetus/hadoop:8f850b46a0c.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/4/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Debug building on Centos 7

* Added -x option for tracing
  bash script execution.

GauthamBanasandra committed:
Add debug option for test-patch.sh

* Need to trace the commands that
  are executed in test-patch.sh.

GauthamBanasandra committed:
Enable debugging of Docker

* Also turned off git remote
  operations.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 33s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shellcheck  |   0m  0s |  |  Shellcheck was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  31m 39s |  |  trunk passed  |
| -1 :x: |  compile  |   0m 23s | [/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in trunk failed.  |
| +1 :green_heart: |  mvnsite  |   0m 27s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  45m 47s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 15s |  |  the patch passed  |
| -1 :x: |  compile  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  cc  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  golang  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  javac  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 15s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  12m 59s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 20s | [/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 32s |  |  The patch does not generate ASF License warnings.  |
|  |   |  61m 45s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs compile cc mvnsite javac unit golang |
| uname | Linux 06cba7bee071 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 48f3961924d859bd218c8fbf2c117502d2f63923 |
| Default Java | Red Hat, Inc.-1.8.0_292-b10 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/testReport/ |
| Max. process+thread count | 720 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/8/console |
| versions | git=1.8.3.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Remove bash tracing

* Removed trace logging
  of bash.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 31s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  shellcheck  |   0m  1s |  |  Shellcheck was not available.  |
| +0 :ok: |  shelldocs  |   0m  1s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  31m 27s |  |  trunk passed  |
| -1 :x: |  compile  |   0m 23s | [/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in trunk failed.  |
| +1 :green_heart: |  mvnsite  |   0m 26s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  45m 44s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 15s |  |  the patch passed  |
| -1 :x: |  compile  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  cc  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  golang  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  javac  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 15s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  13m  5s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 19s | [/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 30s |  |  The patch does not generate ASF License warnings.  |
|  |   |  61m 47s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs compile cc mvnsite javac unit golang |
| uname | Linux d8d4438d69a3 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / f505210e0df6ba474bfa0465d33428aad865e9c2 |
| Default Java | Red Hat, Inc.-1.8.0_292-b10 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/testReport/ |
| Max. process+thread count | 538 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/9/console |
| versions | git=1.8.3.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Run scl enable

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/10/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 40s |  |  Docker failed to build yetus/hadoop:8f850b46a0.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/10/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Install GCC 9.3.0 separately

* Installing GCC 9.3.0 through
  yum or dnf requires a new
  shell to be started, which is
  causing some issues. Thus,
  we're building GCC 9.3.0
  and using the same.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/11/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  | 222m 26s |  |  Docker failed to build yetus/hadoop:8f850b46a0.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/11/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Fix GCC installation

* Hadn't exported the necessary
  environment variables earlier.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/12/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  | 221m 39s |  |  Docker failed to build yetus/hadoop:8f850b46a0.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/12/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Add GCC artifacts to env vars

* Added the GCC artifacts to
  environment variables.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/13/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 19s |  |  Docker failed to build yetus/hadoop:8f850b46a0.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/13/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Use HOME variable

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/14/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  | 203m 16s |  |  Docker failed to build yetus/hadoop:8f850b46a0.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/14/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Use shell login

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  | 260m 52s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shellcheck  |   0m  0s |  |  Shellcheck was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +0 :ok: |  hadolint  |   0m  0s |  |  hadolint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  33m 54s |  |  trunk passed  |
| -1 :x: |  compile  |   0m 18s | [/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in trunk failed.  |
| +1 :green_heart: |  mvnsite  |   0m 21s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  50m 29s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 13s |  |  the patch passed  |
| -1 :x: |  compile  |   0m 11s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  cc  |   0m 11s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  golang  |   0m 11s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  javac  |   0m 11s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 14s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  15m 44s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 15s | [/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 27s |  |  The patch does not generate ASF License warnings.  |
|  |   | 329m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs hadolint compile cc mvnsite javac unit golang |
| uname | Linux b6f0079f475b 4.15.0-142-generic #146-Ubuntu SMP Tue Apr 13 01:11:19 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 1d53702887a4277eb0bb710df17b9ad15b2a3e9d |
| Default Java | Red Hat, Inc.-1.8.0_292-b10 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/testReport/ |
| Max. process+thread count | 519 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/15/console |
| versions | git=1.8.3.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Trigger rerun

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |  34m 11s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shellcheck  |   0m  0s |  |  Shellcheck was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +0 :ok: |  hadolint  |   0m  0s |  |  hadolint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  30m 30s |  |  trunk passed  |
| -1 :x: |  compile  |   0m 22s | [/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/branch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in trunk failed.  |
| +1 :green_heart: |  mvnsite  |   0m 27s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  44m 51s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 15s |  |  the patch passed  |
| -1 :x: |  compile  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  cc  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  golang  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| -1 :x: |  javac  |   0m 14s | [/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/patch-compile-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 16s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  13m  8s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 18s | [/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-native-client.txt) |  hadoop-hdfs-native-client in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 33s |  |  The patch does not generate ASF License warnings.  |
|  |   |  94m 37s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs hadolint compile cc mvnsite javac unit golang |
| uname | Linux dc3db27ec7e7 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 3c7383ae9defc146f75dc9e22bf38fee78f50586 |
| Default Java | Red Hat, Inc.-1.8.0_292-b10 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/testReport/ |
| Max. process+thread count | 543 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/16/console |
| versions | git=1.8.3.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Remove docker

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
|||| _ Prechecks _ |
| -1 :x: |  maven  |   0m  3s |  |  ERROR: maven was not available.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs compile cc mvnsite javac unit golang |
| uname | Linux asf912.gq1.ygridcore.net 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-agent/workspace/hadoop-multibranch_PR-3014/src/dev-support/bin/hadoop.sh |
| git revision | trunk / 0e45c777a70157eda03bc8249382b0bad7b51401 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/17/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Enable docker

GauthamBanasandra committed:
Use bash instead of sh

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/19/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 54s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  1s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  33m 22s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   2m 55s |  |  trunk passed with JDK Ubuntu-11.0.11+9-Ubuntu-0ubuntu2.20.04  |
| +1 :green_heart: |  compile  |   2m 52s |  |  trunk passed with JDK Private Build-1.8.0_292-8u292-b10-0ubuntu1~20.04-b10  |
| +1 :green_heart: |  mvnsite  |   0m 26s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  15m 37s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 16s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 43s |  |  the patch passed with JDK Ubuntu-11.0.11+9-Ubuntu-0ubuntu2.20.04  |
| +1 :green_heart: |  cc  |   2m 43s |  |  the patch passed  |
| +1 :green_heart: |  golang  |   2m 43s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 43s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 47s |  |  the patch passed with JDK Private Build-1.8.0_292-8u292-b10-0ubuntu1~20.04-b10  |
| +1 :green_heart: |  cc  |   2m 47s |  |  the patch passed  |
| +1 :green_heart: |  golang  |   2m 47s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 47s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  3s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/19/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  mvnsite  |   0m 19s |  |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  15m 40s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 107m 19s |  |  hadoop-hdfs-native-client in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 33s |  |  The patch does not generate ASF License warnings.  |
|  |   | 187m 35s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/19/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs hadolint compile cc mvnsite javac unit golang |
| uname | Linux 1f09aa166003 4.15.0-142-generic #146-Ubuntu SMP Tue Apr 13 01:11:19 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 5a7846cbb127325b5a50a6a204e21b0eb1d4eab4 |
| Default Java | Private Build-1.8.0_292-8u292-b10-0ubuntu1~20.04-b10 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.11+9-Ubuntu-0ubuntu2.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_292-8u292-b10-0ubuntu1~20.04-b10 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/19/testReport/ |
| Max. process+thread count | 606 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/19/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Use double quotes

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/20/console in case of problems.

GauthamBanasandra committed:
Use Dockerfile for Centos 7

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/21/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 27s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  |  The patch appears to include 6 new or modified test files.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  35m 14s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   2m 53s |  |  trunk passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  compile  |   2m 59s |  |  trunk passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  mvnsite  |   0m 30s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  13m 17s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 16s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 58s |  |  the patch passed with JDK Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04  |
| +1 :green_heart: |  cc  |   2m 58s |  |  the patch passed  |
| +1 :green_heart: |  golang  |   2m 58s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 58s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 54s |  |  the patch passed with JDK Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08  |
| +1 :green_heart: |  cc  |   2m 54s |  |  the patch passed  |
| +1 :green_heart: |  golang  |   2m 54s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 54s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  4s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/20/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  mvnsite  |   0m 18s |  |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m 21s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  32m 26s |  |  hadoop-hdfs-native-client in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 37s |  |  The patch does not generate ASF License warnings.  |
|  |   | 110m  5s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/20/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3014 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs hadolint compile cc mvnsite javac unit golang |
| uname | Linux 334ded7dc174 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 769411e0935ad6547b0e4573446fecf7034fb7e9 |
| Default Java | Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.10+9-Ubuntu-0ubuntu1.20.04 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_282-8u282-b08-0ubuntu1~20.04-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/20/testReport/ |
| Max. process+thread count | 544 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-native-client U: hadoop-hdfs-project/hadoop-hdfs-native-client |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/20/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Debug Jenkinsfile

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3014/22/console in case of problems.

GauthamBanasandra commented:
Abandoning this. Will be handled in https://github.com/apache/hadoop/pull/3044.
